client/lib/plans/constants.js

```diff
@@ -421,6 +424,16 @@ export const featuresList = {
 		plans: [ PLAN_BUSINESS ]
 	},
 
+	[ FEATURE_LIVE_COURSES ]: {
+		getSlug: () => FEATURE_LIVE_COURSES,
+		getTitle: () => i18n.translate( 'Attend live courses' ),
+		getDescription: () => i18n.translate(
+			"Attend live courses led by Happiness Engineers to get the most out of your site."
```

Comment:
There's a lint error here - we should use single-quotes to define strings unless there's a single-quote contained in the string